### PR TITLE
Refactor local operator tests

### PR DESCRIPTION
**Pull request by PhilipVinc**
Coming soon: A rewrite of the front-end of LocalOperator to speed it up and to make it more amenable to being jax-aware.

However, I need to slightly refactor tests... so here it is. Doing it before hand

### CODE REVIEW

File: test/operator/test_operator.py [hunk 1]
```diff
@@ -120,15 +120,14 @@ def test_produce_elements_in_hilbert(op, attr):
     "op", [pytest.param(op, id=name) for name, op in operators.items()]
 )
 def test_is_hermitean(op):
-    rng = nk.jax.PRNGSeq(0)
+    rng = nk.jax.PRNGSeq(20)
```
Comment (femtobit) [inline test/operator/test_operator.py:123]: Why 20? (I guess "why not" would be a legitimate answer...)

**Review by femtobit — APPROVED** [review]

File: test/operator/test_operator.py (hunk 1 above)
Comment (PhilipVinc) [reply test/operator/test_operator.py:123]: i just wanted a diffrent one from the other test set...

Comment (gcarleo) [reply test/operator/test_operator.py:123]: beautiful, hopefully it doesn't break for seed 21

Comment (PhilipVinc) [reply test/operator/test_operator.py:123]: try and find out...
